pyproject.toml

```diff
@@ -39,3 +40,35 @@ build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
 # write_to = "pyleco/_version.py"
+
+[tool.black]
+line-length = 100
+
+[tool.isort]
+profile = "black"
```

Comment:
do we need those if we are using ruff?

Reply:
I left them in, in case someone wants to use black to format the code.